<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartWidgetClass</class>
 <widget class="QWidget" name="ChartWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="topMargin">
      <number>7</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QLineEdit" name="editMinX">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>10000</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelX">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="editMaxX">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>7</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QLineEdit" name="editMaxY">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frameLastValue">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>0</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Last value:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="editLastValue">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>18</y>
          <width>60</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelY">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frameDeviation">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="editDeviation">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>17</y>
          <width>60</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>2</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Deviation:</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="editMinY">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frameChartPlaceholder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>editMaxY</sender>
   <signal>returnPressed()</signal>
   <receiver>ChartWidgetClass</receiver>
   <slot>MaxYenterPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editMaxY</sender>
   <signal>editingFinished()</signal>
   <receiver>ChartWidgetClass</receiver>
   <slot>MaxYenterPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editMinY</sender>
   <signal>returnPressed()</signal>
   <receiver>ChartWidgetClass</receiver>
   <slot>MinYenterPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editMinY</sender>
   <signal>editingFinished()</signal>
   <receiver>ChartWidgetClass</receiver>
   <slot>MinYenterPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editMinX</sender>
   <signal>returnPressed()</signal>
   <receiver>ChartWidgetClass</receiver>
   <slot>MinXenterPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editMinX</sender>
   <signal>editingFinished()</signal>
   <receiver>ChartWidgetClass</receiver>
   <slot>MinXenterPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editMaxX</sender>
   <signal>returnPressed()</signal>
   <receiver>ChartWidgetClass</receiver>
   <slot>MaxXenterPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editMaxX</sender>
   <signal>editingFinished()</signal>
   <receiver>ChartWidgetClass</receiver>
   <slot>MaxXenterPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MaxYenterPressed()</slot>
  <slot>MinYenterPressed()</slot>
  <slot>MinXenterPressed()</slot>
  <slot>MaxXenterPressed()</slot>
 </slots>
</ui>
